RELEASE_NOTES.md

```diff
@@ -0,0 +1,118 @@
+## WAMR-X.Y.Z
+
+### Breaking Changes
+
+### New Features
+
+### Bug Fixes
+
+### Performance Improvements
```

Comment:
How about changing to `Enhancements`? It might include performance, footprint and some feature enhancements.